packages/generator-common/src/file-writer/package-json.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @internal
+ */
+export interface PackageJsonOptions {
+  npmPackageName: string;
+  version: string;
+  sdkVersion: string;
+  description: string;
+  license?: string;
+}
+
+export function packageJsonBase(
+  options: PackageJsonOptions
+): Record<string, any> {
+  return {
+    name: options.npmPackageName,
+    version: options.version,
+    description: options.description,
```

Comment:
This should have a sensible default value: `An [OData/OpenAPI] client for <packageName> generated by the SAP Cloud SDK v<SDK VERSION>`

Reply:
The two generators provide their description. What would be the value to have a default there, if it is not used under any circumstance?

Reply:
added issue to clean this up: https://github.com/SAP/cloud-sdk-backlog/issues/604